mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/CompletionRecord.java

```diff
@@ -0,0 +1,69 @@
+package org.motechproject.nms.mobileacademy.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Tracks the completion record for a given calling number
+ */
+@Entity(tableName = "nms_ma_completion_records")
+public class CompletionRecord {
```

Comment:
Don't have a need to access any of the auto-gen fields from MDS. Do you still want me to extend it?